coordinator/tests/services/translation/request/dispense/release.spec.ts

```diff
@@ -6,37 +6,44 @@ import {
 } from "../../../../../src/services/translation/request/dispense/release"
 import pino from "pino"
 import {createAuthorForUnattendedAccess} from "../../../../../src/services/translation/request/agent-unattended"
-
+import {createAuthorForAttendedAccess} from "../../../../../src/services/translation/request/agent-attended"
 const logger = pino()
 
 jest.mock("../../../../../src/services/translation/request/agent-unattended", () => ({
   createAuthorForUnattendedAccess: jest.fn()
 }))
 
+jest.mock("../../../../../src/services/translation/request/agent-attended", () => ({
+  createAuthorForAttendedAccess: jest.fn()
+}))
+
 describe("translateReleaseRequest", () => {
   const mockAuthorResponse = new hl7V3.Author()
   mockAuthorResponse.AgentPerson = new hl7V3.AgentPerson()
-  const mockAuthorFunction = createAuthorForUnattendedAccess as jest.Mock
+  const mockUnattendedAuthorFunction = createAuthorForUnattendedAccess as jest.Mock
+  const mockAttendedAuthorFunction = createAuthorForAttendedAccess as jest.Mock
 
```

Comment:
think it'd be good to see a beforeEach that clears all mocks here, I think we're good as is but anywhere we are setting/resetting mocks it's good to make sure we are clearing them too